common/djangoapps/student/management/commands/bulk_change_enrollment_csv.py

```diff
@@ -42,80 +45,126 @@ def add_arguments(self, parser):
         """ Add argument to the command parser. """
         parser.add_argument(
             '--csv_file_path',
-            required=True,
+            required=False,
             help='Csv file path'
         )
+        parser.add_argument(
+            '--file_from_database',
+            default=True,
+            help='Use file from the BulkChangeEnrollmentConfiguration model instead of the command line.',
+        )
+
+    def get_file_from_database(self):
+        """ Returns an options dictionary from the current SSPVerificationRetryConfig model. """
+
+        enrollment_config = BulkChangeEnrollmentConfiguration.current()
+        if not enrollment_config.enabled:
+            raise CommandError('BulkChangeEnrollmentConfiguration is disabled or empty, '
+                               'but --file_from_database from was requested.')
+
+        return enrollment_config.csv_file
 
     def handle(self, *args, **options):
         """ Main handler for the command."""
-        file_path = options['csv_file_path']
+        file_path = options.get('csv_file_path', None)
+        file_from_database = options['file_from_database']
 
-        if not path.isfile(file_path):
-            raise CommandError("File not found.")
+        if file_path:
+            if not path.isfile(file_path):
+                raise CommandError("File not found.")
 
-        with open(file_path) as csv_file:
-            course_key = None
-            user = None
-            file_reader = csv.DictReader(csv_file)
-            headers = file_reader.fieldnames
+            with open(file_path, 'rb') as csv_file:
+                self.change_enrollments(csv_file)
 
-            if not ('course_id' in headers and 'mode' in headers and 'user' in headers):
-                raise CommandError('Invalid input CSV file.')
+        elif file_from_database:
```

Comment:
Can we move this check at the top? Because most of the time we are going to run this job with `--file_from_database` and we can ignore that extra check of file argument.